enter.sh

```diff
@@ -1,6 +1,6 @@
 
 ./do venv
-source ./venv.dir/bin/activate
+. ./venv.dir/bin/activate
```

Comment:
No action required, just looked this up and I realized I had this backwards the whole time. I always thought `.` was the bash-ism, and `source` was the POSIX command.

Reply:
I can never remember, apparently!